tests/test_sbomnix.py

```diff
@@ -12,7 +12,6 @@
 import shutil
 from pathlib import Path
 import json
-import imghdr
```

Comment:
It's imghdr still used elsewhere, or can we remove some dependencies?

Reply:
imghdr is not used anywhere after this change. Which dependencies you had in mind?